src/model.jl

```diff
@@ -609,14 +676,13 @@ function _build_model(
     # Positional indices from mpc.gen
     PMAX = 9
     PMIN = 10
-
     println("building sets: ", Dates.now())
     # Sets - time periods
     hour_idx = 1:interval_length
     end_index = start_index + interval_length - 1
     # Sets - static
-    sets = _make_sets(case, storage)
-
+    print(123)
```

Comment:
This is probably test code that can be removed?

Reply:
> This is probably test code that can be removed?

right.